src/allomancy/java/leaf/cosmere/allomancy/common/manifestation/AllomancyBrass.java

```diff
@@ -27,10 +34,35 @@ public AllomancyBrass(Metals.MetalType metalType)
 
 	//Dampens Emotions
 	@Override
-	protected void applyEffectTick(ISpiritweb data)
+	public void applyEffectTick(ISpiritweb data)
+	{
+		if (data.getLiving().level.isClientSide())
+		{
+			boolean isSingleTarget = Keybindings.MANIFESTATION_USE_ACTIVE.isDown() || AllomancyKeybindings.ALLOMANCY_SOOTHE.isDown();
+			int singleTargetEntityId = 0;
+
+			if (isSingleTarget)
+			{
+				HitResult ray = PlayerHelper.pickWithRange(data.getLiving(), (int) Math.floor(getRange(data) * CosmereConfigs.SERVER_CONFIG.EMOTIONAL_POWERS_SINGLE_TARGET_RANGE_MULTIPLIER.get()));
+				if (ray instanceof EntityHitResult entityHitResult)
+				{
+					singleTargetEntityId = entityHitResult.getEntity().getId();
+				}
+			}
+
+			Allomancy.packetHandler().sendToServer(new EntityAllomancyActivateMessage(Metals.MetalType.BRASS, isSingleTarget, singleTargetEntityId));
```

Comment:
This will be sending a packet every single tick, ie 20 times a second, which is not ideal. can we reduce that down somehow? Though that may run into similar issues as iron/steel, where packets get lost and the server doesn't know to stop any effect.

Reply:
I'll do an isActiveTick check. We might want it faster though, in which case, I'll take any suggestions :)